src/screenshot/screenshot-compare.ts

```diff
@@ -130,7 +130,17 @@ export async function compareScreenshot(
 
 async function getMismatchedPixels(pixelmatchModulePath: string, pixelMatchInput: d.PixelMatchInput) {
   return new Promise<number>((resolve, reject) => {
-    const timeout = jasmine.DEFAULT_TIMEOUT_INTERVAL * 0.5;
+    /**
+     * When using screenshot functionality in a runner that is not Jasmine (e.g. Jest Circus), we need to set a default
+     * value for timeouts. There are issues runtime errors that occur if we attempt to use optional chaining + nullish
```

Comment:
Small grammatical error, occurs in a few other spots as well where this comment was copy-pasta'd

```suggestion
     * value for timeouts. There are runtime errors that occur if we attempt to use optional chaining + nullish
```

Reply:
Bah - the worst part of copy/paste!